### ISSUE TITLE
Get-HPOVStorageSystem default format-table errors

### ISSUE THREAD

**Issue by scriptengine** [HewlettPackard/POSH-HPOneView#218]
The domain and model attributes appear to be swapped in the default table format for Get-HPOVStorageSystem

### Steps to reproduce
(From the Synergy DCS):
```PowerShell
10:20 PS> Get-HPOVStorageSystem

Name              State      Status SerialNumber **Model              DomainName**   WWN                     Firmware
ThreePAR7200-6710 Configured OK     TXQ1000307   **TestDomain         HP_3PAR 7200** 20:00:00:02:AC:00:08:D5 3.2.1.292
ThreePAR7200-6955 Configured OK     TXQ1010307   **ScaleTestingDomain HP_3PAR 7200** 20:00:00:02:AC:00:08:F6 3.2.1.292

#Now list out the attributes specifically:
10:20 PS> Get-HPOVStorageSystem |
ft Name,State,Status,serialNumber,**model,managedDomain**,wwn,firmware

name              state      status serialNumber model        managedDomain      wwn                     firmware
----              -----      ------ ------------ -----        -------------      ---                     --------
ThreePAR7200-6710 Configured OK     TXQ1000307   **HP_3PAR 7200 TestDomain**         20:00:00:02:AC:00:08:D5 3.2.1.292
ThreePAR7200-6955 Configured OK     TXQ1010307   **HP_3PAR 7200 ScaleTestingDomain** 20:00:00:02:AC:00:08:F6 3.2.1.292


10:20 PS> Get-HPOVStorageSystem | ft

Name              State      Status SerialNumber Model              DomainName   WWN                     Firmware
----              -----      ------ ------------ -----              ----------   ---                     --------
ThreePAR7200-6710 Configured OK     TXQ1000307   TestDomain         HP_3PAR 7200 20:00:00:02:AC:00:08:D5 3.2.1.292
ThreePAR7200-6955 Configured OK     TXQ1010307   ScaleTestingDomain HP_3PAR 7200 20:00:00:02:AC:00:08:F6 3.2.1.292
```
### Version Information
PowerShell Library Version : 3.0.1191.2858
Library Path               : C:\Program Files\WindowsPowerShell\Modules\HPONeView.300

**Comment by ChrisLynchHPE** [comment]
Thanks for reporting this.  I have it fixed and will be in the next update.

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 3.0.1210.3013](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v3.0.1210.3013).

Closing.
